src/libraries/System.Private.CoreLib/src/System/IO/TextWriter.cs

```diff
@@ -1018,5 +1021,508 @@ public override Task FlushAsync(CancellationToken cancellationToken)
                 return Task.CompletedTask;
             }
         }
+
+        /// <summary>
+        /// Creates an instance of <see cref="TextWriter"/> that writes supplied inputs to each of the writers in <paramref name="writers"/>.
+        /// </summary>
+        /// <param name="writers">The <see cref="TextWriter"/> instances to which all operations should be broadcast (multiplexed).</param>
+        /// <returns>
+        /// An instance of <see cref="TextWriter"/> that writes supplied inputs to each of the writers in <paramref name="writers"/>
+        /// </returns>
+        /// <exception cref="ArgumentNullException"><paramref name="writers"/> is <see langword="null"/>.</exception>
+        /// <remarks>
+        /// <para>
+        /// The resulting instance will delegate each operation to each of the writers in <paramref name="writers"/>.
+        /// For example, calling <see cref="Write(char)"/> will write the specified char to each writer, one after the
+        /// other. The writers will be written to in the same order as they are specified in <paramref name="writers"/>.
+        /// An exception from the operation on one writer will prevent the operation from being performed on subsequent writers.
+        /// </para>
+        /// <para>
+        /// <see cref="Encoding"/> will return the corresponding object from first writer in <paramref name="writers"/>.
+        /// </para>
+        /// </remarks>
+        public static TextWriter CreateBroadcasting(params TextWriter[] writers)
+        {
+            ArgumentNullException.ThrowIfNull(writers);
+
+            return writers.Length != 0 ?
+                new BroadcastingTextWriter([.. writers]) :
+                Null;
+        }
+
+        private sealed class BroadcastingTextWriter : TextWriter
+        {
+            private readonly TextWriter[] _writers;
+
+            public BroadcastingTextWriter(TextWriter[] writers)
+            {
+                Debug.Assert(writers is { Length: > 0 });
+                foreach (TextWriter writer in writers)
+                {
+                    ArgumentNullException.ThrowIfNull(writer, nameof(writers));
+                }
+
+                _writers = writers;
+            }
+
+            public override Encoding Encoding => _writers[0].Encoding;
```

Comment:
Is this meaningful?

Reply:
Can you clarify your question? What would you expect to happen instead? At least by specifying it uses the encoding from the first, someone can choose to order the writers appropriately. If the encodings differ between the writers and the consumer actually uses the encoding, there's no value here that would "just work".

Reply:
My question is how the get-only `Encoding` property is supposed to be used. It seems that no functionality of the broadcasting writer will be affected.

Reply:
It's a very rarely used property that a consumer can use if they need to generate bytes in the same encoding that the writer will use.